### ISSUE TITLE
2022/04/05 progress

### ISSUE THREAD

**Issue by biancong1208** [ZhengPeiRu21/mod-playerbots#31]
HI, @ZhengPeiRu21 , thanks for your hard work.
This is the result of today's compilation: Compilation failed

![image](https://user-images.githubusercontent.com/93648230/161768085-f29e5a52-7467-4949-83a2-2cc841add063.png)

**Comment by htc16** [comment]
maybe you are using the old mod or wrong branch, because here for me everything is normal.

**Comment by biancong1208** [comment]
> maybe you are using the old mod or wrong branch, because here for me everything is normal.

OK, let me try again.

**Comment by biancong1208** [comment]
> maybe you are using the old mod or wrong branch, because here for me everything is normal.

Sadly, I recompiled and it still failed, 

![image](https://user-images.githubusercontent.com/93648230/161782424-88f7613f-c7d1-4644-94f2-fbb2573ac38e.png)

![azerothcore-wotlk-Playerbot](https://user-images.githubusercontent.com/93648230/161782465-3804cdce-2035-4392-93cf-6eca4412de49.png)

![mod-playerbots-AzerothCore](https://user-images.githubusercontent.com/93648230/161782491-65a2f541-225c-472d-ae3b-2415fd1d63f2.png)

**Comment by htc16** [comment]
try to do everything manually, download the playerbots branch and then the playerbots mod and put it manually in the modules folder.  there is no longer the term "mod-playerbots-AzerothCore".  And that's even why I'm thinking you might be getting confused and using the old mod.

**Comment by ZhengPeiRu21** [comment]
The correct directory name should be `mod-playerbots`, not `mod-playerbots-AzerothCore`. Because CMake uses special defines for this directory name, it is necessary to use this specific name. Can you please try changing the directory name and see if this resolves your compile issues?

**Comment by biancong1208** [comment]
It's really a bit strange. I changed the names of the two folders back to the original ones, and the compilation was successful. The strange thing is that these two names were used before, but the compilation failed.
![2022-04-06_204436](https://user-images.githubusercontent.com/93648230/161977637-b8937ba8-e992-47c2-976d-1ef7cbd5accb.png)
I can finally start playing, I am so happy, I have to go and experience it.
